app/webpacker/styles/stories.scss

```diff
@@ -1,19 +1,17 @@
 .stories-feature {
-
-    display: table;
+    display: flex;
```

Comment:
Is it worth setting an ie11 fallback since that doesn't support `display: flex`, eg 

```css
display: -ms-flexbox;
display: flexbox;
```

A lot of DfE staff will view the pages using the default browser on their computers which is (certainly used to be) IE11